manuscript/draft.Rmd

```diff
@@ -273,6 +273,54 @@ Looked at how error changed as the time horizon of forecasting lengthened
 
 ## Results
 
+The mixed model (which formed the basis of our observer-corrected "average"
+predictions) found that 70% of the variance in richness in the training set
+could be explained by differences among sites, and 21% could be explained by
+differences among observers. The remaining 9% represents residual variation,
+where a given observer might report a different number of species on different
+runs of the same transect. We found remarkably little autocorrelation in these
+residuals ([[about 0.04, +/- something]]), suggesting richness is close to
+stationary on these time scales, once observer differences are accounted for.
+
+The substantial uncertainty associated with differences among observers caused
+reliable improvements in both mean-squared-error and deviance when observer
+effects were controlled for (Figure 4). This improvement was smallest for the
+"Naive" model, as discussed below.
```

Comment:
I think this paragraph could use a bit more detail in relation to our discussions yesterday. We should talk about how it results in an improvement in the average RMSE and deviance across the board, but that in some cases this is driven more by outliers than the body of the distribution. We should note that even when this is the case it means that the number of really poor forecasts is decreased.

"reliable improvements" indicates to me that that there are consistent improvements across sites. I'd recommend rephrasing during the expansion to avoid confusion.